CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## v0.11.0
+
+## Added
+- Added `rand_bytes` function that generates a pseudo random vector of
+  `len` bytes.
+- Added implementation of `std::error::Error` for `fam::Error`.
```

Comment:
We should mention about https://github.com/rust-vmm/vmm-sys-util/commit/10a5ac19a0a898076d94512e633abfb010d0fb4d as well.

Reply:
Good catch, I'll add it.